src/runner/browser-job.ts

```diff
@@ -37,13 +41,14 @@ export default class BrowserJob extends AsyncEventEmitter {
     public readonly warningLog: WarningLog;
     public readonly fixtureHookController: FixtureHookController;
     private _result: BrowserJobResultInfo | null;
-    private readonly _testRunControllerQueue: TestRunController[];
+    private _testRunControllerQueue: TestRunController[];
     private readonly _reportsPending: TestRunController[];
     private readonly _connectionErrorListener: (error: Error) => void;
     private readonly _completionQueue: TestRunController[];
     private _resolveWaitingLastTestInFixture: Function | null;
     private readonly _messageBus: MessageBus;
     private readonly _testRunHook: TestRunHookController;
+    private readonly _noConcurrencyQueue: NoConcurrencyQueue;
```

Comment:
`NoConcurrencyQueue` is unnecessary. Just make it like this `Dictionary<TestRunController[]>` and remove interface `NoConcurrencyQueue`